README-FOR-WAGTAIL-LOCALISATION.md

```diff
@@ -18,7 +18,7 @@ The modeltranslation library adds localised URLs to Wagtail, interjecting the lo
 
 with "en" as locale, becomes:
 
-    https://foundation.mozilla.org/en/campaigns/aadhaar
+    https://foundation.mozilla.org/campaigns/aadhaar
```

Comment:
this one needs to stay, as the text literally explains that hitting a URL without a locale will inject one =)